inference-engine/tests/functional/plugin/shared/include/subgraph_tests/matmul_squeeze_add.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+typedef std::tuple<
+    InferenceEngine::Precision,          // Network Precision
+    std::string,                         // Target Device
+    std::map<std::string, std::string>   // Configuration
+> matmulSqueezeAddParams;
```

Comment:
Better make this test with more varied input and constants sizes. Some being aligned some unaligned. to have better coverage